inji-web/src/components/Help/HelpAccordion.tsx

```diff
@@ -1,51 +1,129 @@
 import React, {useState} from "react";
 import {HelpAccordionItem} from "./HelpAccordionItem";
+import { HelpAccordionItemProps } from "../../types/components";
 import {useTranslation} from "react-i18next";
+import { title } from "process";
 
 export const HelpAccordion: React.FC = () => {
 
     const [open, setOpen] = useState(0);
     const {t} = useTranslation("Help");
+
+    const createContent = (key: string, descriptions: string[]) => {
+        return descriptions.map((desc, index) => {
+            if (key === "item21" || key ==="item22") {
```

Comment:
Do no hardcode any Keys ? make the method more generic, so that you don't have the touch the method again.

Either, you can check if the description is valid html or not
( or ) you can add another param which lets u decide if you want to render as string or html

you can remove **key** param as it not needed.